.circleci/config.yml

```diff
@@ -7,15 +7,17 @@ jobs:
     working_directory: ~/code
     docker:
       # CircleCI Python images available at: https://hub.docker.com/r/circleci/python/
-      - image: python:3.7
+      - image: circleci/python:3.7-node-browsers
         environment: # environment variables for primary container
           PIPENV_VENV_IN_PROJECT: true
           DATABASE_URL: postgresql://root@localhost/circle_test?sslmode=disable
+          NODE_ENV: development # Only needed if puppeteer is in `devDependencies`
```

Comment:
@alexsoble do we still need this?

Reply:
Hmmmmm — let's try removing it and see if the tests still pass!